spec/features/budgets/investments_spec.rb

```diff
@@ -8,7 +8,7 @@
   let(:budget)  { create(:budget, name: "Big Budget") }
   let(:other_budget) { create(:budget, name: "What a Budget!") }
   let(:group) { create(:budget_group, name: "Health", budget: budget) }
-  let!(:heading) { create(:budget_heading, name: "More hospitals", price: 666666, group: group) }
+  let!(:heading) { create(:budget_heading, name: "More hospitals", price: 666666, group: group, latitude: '40.416775', longitude: '-3.703790') }
```

Comment:
If the latitude and longitude of a heading has already default values defined in the factory, they should not need to be added to the specs again, unless it's something relevant for that particular spec.

Regarding relevant specs, I'm missing a new `context`, say "Map in sidebar", with a few `scenarios` specifically testing this funcionality. For e.x. given 5 investments coordinates all over New York, when I visit Manhattan I should only see in the map the Manhattan investments.

Could you remove the unneeded `latitude` and `longitude` parameters for headings in specs 😌and add specific scenarios for the map? 🙏

Reply:
I've added some test scenarios related to the map in sidebar.

Reply:
Thank you @MatheusMiranda! 👍